gradle/lib/dependabot/gradle/file_updater/wrapper_updater.rb

```diff
@@ -84,8 +100,12 @@ def target_file?(file)
         def command_args
           version = T.let(dependency.requirements[0]&.[](:requirement), String)
           checksum = T.let(dependency.requirements[1]&.[](:requirement), String) if dependency.requirements.size > 1
+          source = T.let(dependency.requirements[0]&.[](:source), T::Hash[Symbol, String])
+          distribution_url = source.[](:url)
+          distribution_type = distribution_url&.match(/\b(bin|all)\b/)&.captures&.first
```

Comment:
Are we sure that this is needed? My understanding is that gradle would detect this bu default

Reply:
I that my understanding as well, but empiric evidence tells otherwise. It will revert back to `bin` unless we tell otherwise.